app/views/govuk_publishing_components/components/feedback/_yes_no_banner.html.erb

```diff
@@ -1,17 +1,24 @@
-<%
-  contact_govuk_path = "/contact/govuk"
-%>
-
 <div class="gem-c-feedback__prompt gem-c-feedback__js-show js-prompt" tabindex="-1">
   <div class="gem-c-feedback__prompt-questions js-prompt-questions" hidden>
     <h2 class="gem-c-feedback__prompt-question"><%= t("components.feedback.is_this_page_useful") %></h2>
     
     <ul class="gem-c-feedback__option-list">
-      <li class="gem-c-feedback__option-list-item" style="display: none" hidden="hidden">
-        <!-- Maybe button exists only to try and capture clicks by bots -->
-        <button data-track-category="yesNoFeedbackForm" data-track-action="ffMaybeClick" aria-expanded="false" style="display: none" hidden="hidden" aria-hidden="true">
+      <li class="gem-c-feedback__option-list-item govuk-visually-hidden" style="display: none" hidden>
+        <% # Maybe button exists only to try and capture clicks by bots %>
```

Comment:
This whole trap is a bit confusing to me as I've never been clear on why we expect bots to target this button rather than another... but that aside, why do you think turning it into a link will aid here? I'm also aware that it lands you on a page that contains a form - do we risk sending the nasty traffic away from our form in favour of spamming this one instead?